tests/console/w3m.pm

```diff
@@ -0,0 +1,47 @@
+# SUSE's openQA tests - FIPS tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Case 1525204 - FIPS: w3m
+
+use base "consoletest";
+use strict;
+use testapi;
+
+sub run() {
+    my $name = ref(@_[0]);
```

Comment:
where did you copy-paste this from? I don't see you using any variable and this construct looks weird.